image.yaml

```diff
@@ -147,6 +147,13 @@ modules:
     - name: activemq.artemis.s2i
     - name: amq.custom.config
 
+artifacts:
+    - name: "jolokia-javaagent-jar"
+      url: https://repo1.maven.org/maven2/org/jolokia/jolokia-agent-jvm/2.0.3/jolokia-agent-jvm-2.0.3-javaagent.jar
+      md5: "afaf573d1ca5457771375cb045ebe6bd"
+      dest: /opt
```

Comment:
The file `jolokia-javaagent-jar` could be in a subdirectory of `opt` when mounting a file is not easy.

Reply:
done, thanks. 
can now referenced via -javaagent:/opt/jolokia/javaagent.jar